rigs/heft-node-rig/README.md

```diff
@@ -0,0 +1,18 @@
+## @rushstack/heft-node-rig
+
+This is a Rig package for [Heft](https://www.npmjs.com/package/@rushstack/heft)
+used for building NodeJS projects.
+
+This rig contains a single profile: `library`
```

Comment:
What do we use for CLI tools like Rush?

My company also builds Node.js web services (although today they use pretty much the same config as a tool)

Reply:
CLI tools aren't really different from any other Node library.

Web services will probably need their own profile.